lib/utils/index.js

```diff
@@ -1,6 +1,8 @@
 const handlebars = require('handlebars');
+const falsey = require('falsey');
```

Comment:
I spent a lot of time reducing the number of dependencies this library has to zero. Can we please do the same here?